### ISSUE TITLE
Aspect Ratio gets set incorrectly

### ISSUE THREAD

**Issue by demisjohn** [demisjohn/Microscope-Measurement-Tools#12]
Looks like the Y-PixelHeight is being set incorrectly, working on it.

When calling old calibrations that used to work fine, now see this:
<img width="281" alt="Set Scale window" src="https://user-images.githubusercontent.com/5370181/56054720-6bbc3d00-5d0c-11e9-80a6-7d178debcc83.png">
Incorrect Aspect Ratio - should say 1.0 from the Settings file.

**Comment by demisjohn** [comment]
PixelHeight is always set to 1.0, for some reason.
Tried enforcing floating-point but it didn't fix it.
Console:
```
Chose Cal ` Microscope #2 (Nikon): 150x ` :  28.862  px/ um
('newAspect = ', 1.0, ', newPixelWidth=', 0.0346476335666274, ', newPixelHeight =', 1.0, <type 'float'>)
```

`newPixelHeight` is supposed to be the same as `newPixelWidth`, due to the math:
`newPixelHeight = newcal.pixelWidth * newAspect`

Maybe it get's stomped later in the script?

**Comment by demisjohn** [comment]
Found typo.
Fixed in a3fb47cbbcd2eeb5fd62434cd052cea4bff4a46a
